.github/workflows/main.yml

```
@@ -0,0 +1,26 @@
+name: Main testing workflow
+
+on: [push]
```

This will not run the `main` workflow on PR created from forks.
Also sometimes it's really handy to run workflow manually - on the `workflow_dispatch` event.

```suggestion
on:
  push:
  pull_request:
  workflow_dispatch:
```

Please note that  with the proposed change workflow will run twice for all PRs created from the `pytest-factoryboy` branch.